platforms/storybook/src/stories/Popup.stories.tsx

```diff
@@ -1,18 +1,30 @@
 import React from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 import Popup from '@shared/extension/popup/components/Popup';
+import { createTheme, ThemeProvider } from '@material-ui/core/styles';
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const ytLogo = require('../../../yttrex/extension/public/yttrex-logo.png');
 
+// define @mui theme for popup
+const theme = createTheme({
+  typography: {
+    fontFamily: 'Trex-Regular',
```

Comment:
I think is better to have it defined [here](https://github.com/tracking-exposed/trex/blob/master/platforms/storybook/.storybook/preview.js), so it works for every story we have and we create